README.md

```diff
@@ -8,103 +8,67 @@
+```
+curl -XPOST \
+     -H 'Content-Type: application/json' \
+     -H 'Micro-Endpoint: Helloworld.Greeting' \
+     -d '{"name": "alice"}' \
+      http://localhost:8080
+```
 
+See the [samples](https://github.com/apache/dubbo-go-samples) for detailed information on usage. Next, learn how to [deploy](), [monitor]() and [manage the traffic]() of your dubbo-go application.
 
-### Features
-|      Feature       | IDEA | GoLand |
-|:------------------:|:----:|:------:|
-| Hessian2 Generator |  ✅️  |   ✅️   |
-| New Project/Module |  ✅️  |   ✅️   |
-
-#### Project/module templates
-| Project/Module Template | Progress |
-|:-----------------------:|:--------:|
-|         Sample          |    ✅️    |
-|      Empty Project      |    ✅️    |
-
-##### Empty project template middleware
-|  Middleware  |                Module                 | Support |
-|:------------:|:-------------------------------------:|:-------:|
-| Web Service  |    [Gin](github.com/gin-gonic/gin)    |   ✅️    |
-| Memory Cache | [Redis](github.com/go-redis/redis/v8) |   ✅️    |
-|   Database   |         [Gorm](gorm.io/gorm)          |   ✅️    |
+## Features
 
+<img src="https://dubbogo.github.io/img/devops.png" height="300px" display="display: block, margin: auto" />
 
-If you want to know more about the Intellij Plugin for Dubbo-go, you may
-refer to [https://gitee.com/changeden/intellij-plugin-dubbo-go-generator](https://gitee.com/changeden/intellij-plugin-dubbo-go-generator).
+- **RPC Protocols**: Triple(gRPC compatible and HTTP-friendly), Dubbo2(TCP)
+- **Service Discovery**: Nacos, Zookeeper, Etcd, Polaris-mesh, Consul.
+- **Load Balance**: Adaptive, Random, RoundRobin, LeastActive, ConsistentHash
+- **Traffic Management**: traffic split, timeout, rate limiting, canary release
+- **Configuration**: yaml file, dynamic configuration(Nacos, Zookeeper, etc.).
+- **Observability**: metrics(Prometheus, Grafana) and tracing(Jaeger, Zipkin).
+- **HA Strategy**: Failover, Failfast, Failsafe/Failback, Available, Broadcast, Forking
 
 ## Ecosystem
-
-* [Dubbo Ecosystem Entry](https://github.com/apache?utf8=%E2%9C%93&q=dubbo&type=&language=) - A GitHub group `dubbo` to gather all Dubbo relevant projects not appropriate in [apache](https://github.com/apache) group yet.
-* [dubbo-go-pixiu](https://github.com/apache/dubbo-go-pixiu) - A dynamic, high-performance API gateway solution for Dubbo and Http services.
-* [dubbo-go-samples](https://github.com/apache/dubbo-go-samples) - Samples for Apache Dubbo-go.
-* [dubbo-getty](https://github.com/apache/dubbo-getty) - A netty like asynchronous network I/O library which supports tcp/udp/websocket network protocol.
-* [triple](https://github.com/dubbogo/triple) - A golang network package that based on http2, used by Dubbo-go 3.0.
-* [dubbo-go-hessian2](https://github.com/apache/dubbo-go-hessian2) - A golang hessian library used by Apache/dubbo-go.
-* [gost](https://github.com/dubbogo/gost) - A go sdk for Apache Dubbo-go.
```

Comment:
I am planning to move them to the website. Since most of them are libraries relied on by dubbo-go, they might be useful for maintainers but not helpful for most users.

Reply:
Or the contributing page